docs/developer-docs/latest/development/typescript.md

```diff
@@ -6,14 +6,10 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/typescrip
 
 # TypeScript development
 
-TypeScript adds an additional type system layer above JavaScript, which means that existing JavaScript code is also TypeScript code. Strapi supports TypeScript in new projects on v4.2.0 and above. TypeScript-enabled projects allow developing plugins with TypeScript as well as using TypeScript typings.
+TypeScript adds an additional type system layer above JavaScript, which means that existing JavaScript code is also TypeScript code. Strapi supports TypeScript in new projects on v4.3.0 and above. Existing JavaScript projects can [add TypeScript support](#add-typescript-support-to-an-existing-strapi-project) through a short migration procedure. TypeScript-enabled projects allow developing plugins with TypeScript as well as using TypeScript typings.
```

Comment:
I'm wondering about the use of "short" 🤔 I tend to think we should avoid it, just like we avoid "simple" or "easy". Especially because if at some point a user has an issue, it could take a longer time, and they might be frustrated. 
Also, as we've just discussed it during the DX sync meeting, maybe "conversion" is better than "migration"? :)